scripts/exrrfs_jedivar.sh

```diff
@@ -166,7 +166,27 @@ cp "${DATA}"/jdiag* "${COMOUT}/jedivar/${WGF}"
 cp "${DATA}"/jedivar*.yaml "${COMOUT}/jedivar/${WGF}"
 cp "${DATA}"/log.out "${COMOUT}/jedivar/${WGF}"
 cp "${DATA}"/mpasin.nc "${COMOUT}/jedivar/${WGF}/mpasout.${timestr}.nc"
-if ls ./satbias_out/*satbias*.nc >/dev/null 2>&1; then
-  cp "${DATA}"/satbias_out/*satbias*.nc "${COMOUT}/jedivar/${WGF}"
+
+# copy satbias files who are not updated in the current cycle to satbias_out/
```

Comment:
who -> which

Reply:
Thanks, changed and will be committed soon.